print.py

```diff
@@ -0,0 +1,102 @@
+import sys
+import os
+import time
+from threading import Lock
+
+from ftxwebsocket.client import FtxWebsocketClient
+from ftxrest.client import FtxClient
+
+
+if len(sys.argv) < 2:
+    raise ValueError("Must specify a market")
+
+MARKET = sys.argv[1]
+
+last_printed_trade = None
+ws_trades = []
+print_lock = Lock()
+rest_client = FtxClient(
+    api_key="Doesnt matter, just using public APIs",
+    api_secret="Doesnt matter, just using public APIs",
+)
+
+
+def _fetch_missing_trades_when_appropriate():
+    last_trade = _get_last_ws_trade()
+
+    with print_lock:
```

Comment:
Use a print lock to serialize trade print attempts occurring on the main thread using trades received via websocket and on the background thread using trades received via REST requests. We similarly acquire this lock before printing on https://github.com/73e255823b/ftx/pull/1/files#diff-7fd86dc6f09e2aeab4cf5081126e12430eba4e9edd42861d82660e5ac5a75d7dR95. This ensures that REST trade information fetched during websocket downtime is printed to stdout before any new websocket-received trade info.